upgrade-prechecks/preupgrade-healthcheck.sh

```diff
@@ -919,42 +928,97 @@ function verify_pid_limit(){
     rm -f ${TEMP_MMHEALTH_FILE} >> /dev/null
 }
 
+function is_metrodr_setup(){
+    metrodrinstances=$(oc get metrodrs -n $FUSIONNS | tail -n +2 | wc -l)
+    if [ $metrodrinstances -gt 0 ]; then
+        echo 1
+    else
+        echo 0
+    fi
+}
+
+function verify_CSI_configmap_present(){
+    print info "Verify presence of CSI configmap if it is a 2.6.1 MetroDR setup"
+    isfversion=$(oc get csv -n ibm-spectrum-fusion-ns | grep -n . | tail -n +2 | head -n 1 | awk '{print $5}' | grep "2.6.1" | wc -l)
```

Comment:
use variable we have for fusion namespace, do nit user directly ibm-spectrum-fusion-ns

Reply:
added.